selfhost/main.jakt

```diff
@@ -154,6 +154,7 @@ function main(args: [String]) {
         let basename_without_extension = file_path.basename().substring(start: 0, length: file_path.basename().length() - 5)
         let cpp_filename = binary_dir + "/" + basename_without_extension + ".cpp"
         let output_filename = binary_dir + "/" + basename_without_extension
+        mkdir(binary_dir)
```

Comment:
This is an unknown function, even though I attempted to add `mkdir` to the prelude. I need help :sweat_smile:

Reply:
You didn't add  `#include <IO/fs.cpp>` to `lib.h`. The runtime is all compiled as one massive translation unit still.

Reply:
Thanks! After adding that, there's a different error. CI shows the same thing as I see when I run it locally:

```
/usr/include/x86_64-linux-gnu/sys/stat.h:317:12: note: candidate function not viable: requires 2 arguments, but 1 was provided
extern int mkdir (const char *__path, __mode_t __mode)
           ^
```

Reply:
yeah I mean, that's what I figured. I copied this from jakttest and was confused when I saw that because it seems to work in jakttest...?

Reply:
Alright, I think the fs.h/fs.cpp thing was incorrect. All I needed to do was have the extern functions listed in prelude.jakt. 

I was getting: `projects/jakt/build/main.cpp:27673:17: error: use of undeclared identifier 'mkdir'`

I fixed that by adding `#include <sys/stat.h>` (though maybe not in the right place)

The only part left is figuring out how to represent the `mode` parameter in jakt